i3pystatus/github.py

```diff
@@ -1,83 +1,544 @@
+    '''
+    settings = (
+        ('format', 'format string'),
+        ('status', 'Dictionary mapping statuses to the text which represents '
+                   'that status type. This defaults to ``GitHub`` for all '
+                   'status types.'),
+        ('colors', 'Dictionary mapping statuses to the color used to display '
+                   'the status text'),
+        ('refresh_icon', 'Text to display (in addition to any text currently '
+                         'shown by the module) when refreshing the GitHub '
+                         'status. **NOTE:** Depending on how quickly the '
+                         'update is performed, the icon may not be displayed.'),
+        ('update_error', 'Value for the ``{update_error}`` formatter when an '
+                         'error is encountered while checking GitHub status'),
+        ('keyring_backend', 'alternative keyring backend for retrieving '
+                            'credentials'),
+        ('username', ''),
+        ('password', ''),
+        ('access_token', ''),
+        ('unread_marker', 'Defines the string that the ``{unread}`` formatter '
+                          'shows when there are pending notifications'),
+        ('notify_status', 'Set to ``True`` to display a desktop notification '
+                          'on status changes'),
+        ('notify_unread', 'Set to ``True`` to display a desktop notification '
+                          'when new notifications are detected'),
+        ('unread_notification_template',
+            'String with no more than one ``%d``, which will be replaced by '
+            'the number of new unread notifications. Useful for those with '
+            'non-English locales who would like the notification to be in '
+            'their native language. The ``%d`` can be omitted if desired.'),
+        ('api_methods_url', 'URL from which to retrieve the API endpoint URL '
+                            'which this module will use to check the GitHub '
+                            'Status'),
+        ('status_url', 'The URL to the status page (opened when the module is '
+                       'double-clicked with the right mouse button'),
+        ('notifications_url', 'The URL to the GitHub notifications page '
+                              '(opened when the module is double-clicked with '
+                              'the left mouse button'),
+    )
+
+    # Defaults for module configurables
+    _default_status = {
+        'good': 'GitHub',
+        'minor': 'GitHub',
+        'major': 'GitHub',
+    }
+    _default_colors = {
+        'good': '#2f895c',
+        'minor': '#f29d50',
+        'major': '#cc3300',
+    }
+
+    # Module configurables
+    format = '{status}[ {unread}][ {update_error}]'
+    status = _default_status
+    colors = _default_colors
+    refresh_icon = '⟳'
+    update_error = '!'
     username = ''
     password = ''
     access_token = ''
-    format = '{unread}'
+    unread_marker = '•'
```

Comment:
Ahh, I previously had only the marker with no text so I wouldn't have noticed if it messed up the alignment. It's fine if you leave that one and I can manually specify the larger one in my config.